smoke-tests/smoke-e2e.bats

```diff
@@ -214,6 +226,14 @@ mk_diag_attr() {
     assert_not_empty $(spans_on_view_named "@honeycombio/instrumentation-uikit" "Touch Ended" "Simple Button")
     assert_not_empty $(spans_on_view_named "@honeycombio/instrumentation-uikit" "Touch Ended" "accessibleButton")
     # UISwitch does not support Touch Ended events at this time. Apple sets the view to null.
+    
+    screen_attr=$(attribute_for_span_key "@honeycombio/instrumentation-uikit" "Touch Began" "screen.name" string | uniq)
+    screen_attr=$(attributes_from_span_named "@honeycombio/instrumentation-uikit" "Touch Began" \
+        | jq "select (.key == \"screen.name\")" \
+        | jq "select (.value.stringValue == \"SwiftUI.UIKitTabBarController/UIKitNavigationRoot/UI KIT SCREEN OVERRIDE\").value.stringValue" \
+        | uniq
+    )
+    assert_equal "$screen_attr" '"SwiftUI.UIKitTabBarController/UIKitNavigationRoot/UI KIT SCREEN OVERRIDE"'
```

Comment:
Just a thought, and please feel free to push back, but do you think we should have `screen.name` as a whole path like this, or do you think it would be better to have `screen.path` and `screen.name` as separate fields where the name is the last segment of the path? I dunno, I don't want to complicate things, so we could just do this and revisit it later. I think the main question is how the name looks in the timeline analysis UI.

Reply:
That sounds reasonable, would match what we get from the manual instrumentation on the SwiftUI side.

Reply:
done in https://github.com/honeycombio/honeycomb-opentelemetry-swift/pull/26/commits/6dd54600a66cdadcbf1c37b019d59445a8e1d9bf